documentation/book/con-custom-resources-status.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// assembly-custom-resources.adoc
+
+[id='con-custom-resources-status-{context}']
+= {ProductName} custom resource status
+
+The `status` property of a {ProductName}-specific custom resource is configured to capture the _current_ state of the resource and make that information available.
+
+Status information is useful for tracking progress related to a resource achieving its _desired_ state, providing the time and reason the state of the resource changed and details of events causing delays.
+
+NOTE: The status field is different from the spec field, which describes the _desired_ state of an object.
+
+{ProductName} creates and maintains the status of custom resources, periodically evaluating the current state of the custom resource and updating its status accordingly.
+
+Here we see the `status` property specified for a Kafka custom resource.
+
+.Kafka custom resource with status
+[source,yaml,subs="attributes+"]
+----
+apiVersion: kafka.strimzi.io/v1beta1
+kind: Kafka
+metadata:
+spec:
+  # ...
+status:
+  conditions: <1>
+  listeners: <2>
+    plain:
+      host: my-cluster-kafka-bootstrap.myproject.svc
+      port: 9092
+    tls:
+      host: my-cluster-kafka-bootstrap.myproject.svc
+      port: 9093
+    external::
+      host: my-cluster-kafka-external-bootstrap.myproject.svc
+      Port: 9094
+    ready: true <3>
+    # ...
+----
+<1> Status `conditions` describe criteria related to the status that cannot be deduced from the existing resource information, or are specific to the instance of a resource.
+<2> The `listeners` describe the current Kafka bootstrap addresses by type.
+<3> The `ready` field indicates whether the resource has been successfully reconciled (approved for traffic by the cluster operator).
+
+.Accessing status information
+You can access status information for a resource from the command line. For more information, see xref:proc-accessing-resource-status-{context}[].
```

Comment:
This comment might not be actionable, but I feel it needs to be made nonetheless: The splitting up of the conceptual and procedural info into different assemblies is really unhelpful to the reader here. We've just explained the structure of the `status` property. Now is the perfect time to show them how to access it and provide the example output to reinforce the initial understanding they got from the concept. Adding a link is just unhelpful because it means they can't just scroll up and down to cross reference.

Reply:
Understood. The concepts fit naturally at this level of the doc, but the procedure might be introduced too soon, hence the divide. If @scholzj doesn't mind seeing the procedure in the getting started, I could move it into the custom resources assembly to sit under the concepts

Reply:
I think it really takes away from the _Getting started_. If we want the _Getting started_ chapter to explain details about CRDs, status sub-resources etc., we should IMHO work more on the _Quickstart_. Because the _Getting started: is getting quite quickly far away from serving as _Quickstart_ on the website as it originally did.

Reply:
We can take a look at the structure again after this release. I'd prefer this content sat together.